libs/api/access-status/api/feature/src/lib/api-access-status.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { Field, ObjectType } from "@nestjs/graphql";
+
+@ObjectType()
+export class ApiAccessStatusEntity {
+    @Field()
+    UserID!: string;
+
+    @Field()
+    item!: string;
+
+    @Field()
+    accessStatus: string;
```

Comment:
```suggestion
    @Field()
    userID!: string;

    @Field()
    item!: string;

    @Field()
    accessStatus: string;
```
I think the class member variables should all follow camelCase

Reply:
This is correct thank you.